### DISCUSSION TITLE
Inconsistency in calculating the CCS of C60+ in He and N2

### DISCUSSION THREAD

**Discussion by CorentinRossi** [HopkinsLaboratory/MobCal-MPI#13] [General]
Good evening,

We have completed all the tests described in the manual to ensure that Mobcal was installed and compiled correctly. The results we obtained matched those you provided.

Additionally, we tested Mobcal against other species, including naphthalene and anthracene, as listed in the database, and the agreement was good. However, for C60+, we consistently observed CCS values that were 15% lower in N2 and 3% lower in He compared to both experimental data and the values calculated using Mobcal version 1.

I have attached a table summarizing these results, as well as all relevant log files from Gaussian, Mobcal input files, and Mobcal output files.

Could you please help us understand the origin of this discrepancy and why it appears specifically for C60+? We have tried several optimizations for C60+ and also used optimized structures from the literature, all of which agree within 1%.

Thank you very much for your assistance.

Best wishes,

[Data_C60_Naph_Anthra.zip](https://github.com/user-attachments/files/17213523/Data_C60_Naph_Anthra.zip)

**Comment by ChristianIeritano** [comment]
This issue likely stems from two factors: 
1. MobCal-MPI was parameterized for protonated species, not radical cations like C60+, and
2. the sp2 carbon's van der Waals (vdW) component in the ion-neutral interaction potential. 

The vdW component of the interaction potential used in Mobcal-MPI (and all other TM-based CCS codes for that matter) are optimized using **mostly** protonated molecules with sp2 carbons being located in common functional groups like carbonyls, alkenes, phenyl rings, etc. It’s not surprising that vdW parameters used to describe these sp2 carbons yield poor results for fullerenes.

You might consider deriving new vdW parameters for sp2 carbons in fullerene-like molecules. These parameters are the MMFF94 vdW parameters as described in:

Halgren, J. Comput. Chem. 1996, 17(5-6), 490-519
Halgren, J. Comput. Chem. 1996, 17(5), 520-552

In the order they appear in the vdW file, they are:

    Alpha: atomic polarizability
    N: Slater-Kirkwood effective number of valence electrons
    A: scaling factor (fixed for periodic table rows)
    G: scaling factor (atomic species dependent)

See Table III of the second paper. Interaction distances and well depths are calculated using Equations 2-5 from the same paper. You would need to calculate these quantities as they pertain to fullerene sp2 carbons, then combine them with the parameters that define N2 via the method [here](https://pubs.rsc.org/en/content/articlelanding/2018/an/c8an00270c#!divAbstract). 

Hope that helps.

**Comment by CorentinRossi** [comment]
Thank you for your detailed explanation. While your proposed solution seems to address some aspects of the issue, it may not fully resolve the first concern, as MobCal-MPI was designed for protonated species rather than radical cations. Additionally, deriving new vdW parameters for sp2 carbons in fullerenes could be quite complex for us to manage.

Would you be able to provide additional guidance or tools to help simplify this process? That would be very helpful.

In the meantime, we will explore using version 1 of the software to see if it helps mitigate the issue.

Thank you in advance for your assistance.

**Comment by ChristianIeritano** [comment]
The two papers I mentioned above describe how the vdW parameters are calculated:

Halgren, J. Comput. Chem. 1996, 17(5-6), 490-519
Halgren, J. Comput. Chem. 1996, 17(5), 520-552

The SI of the following [paper](https://pubs.rsc.org/en/content/articlelanding/2018/an/c8an00270c#!divAbstract) describes how the atom specific vdW parameters they are combined with N2. 

A "cheaty" way to do this without re-parameterization is to realize that for the following vdW parameters: 

Alpha: atomic polarizability
N: Slater-Kirkwood effective number of valence electrons
A: scaling factor (fixed for periodic table rows)
G: scaling factor (atomic species dependent)

Whether the sp2 carbon is from a "biological" moiety or a fullerene, the only thing that changes is the polarizability (Alpha). The Slater-Kirkwood effective number of valence electrons should be the same, and A and G are both scaling factors that are fixed for periodic table rows and atomic species, respectively. 

Thus, all you need to do is change the alpha vdW parameter. Because your atom types are all the same for fullerenes, I would sample alpha parameters until you find a value that returns a CCS that matches the experimental value *for a single molecule (say C60+)*. Then, for all other fullerenes, calculate their CCS using the new alpha parameter you just found. 

Keep me updated on the progress, as this would be useful to include on the next release of MobCal-MPI.
